examples/experimental/torch/mobilenet_v2/main.py

```diff
@@ -0,0 +1,178 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import subprocess
+import time
+from pathlib import Path
+
+import nncf
+import torch
+import torchvision
+from openvino.runtime import Core
+
+from examples.experimental.torch.mobilenet_v2 import utils
+
+# Path to the `mobilenet_v2` directory.
+ROOT = Path(__file__).parent.resolve()
+# Path to the directory where the original and quantized models will be saved.
+MODEL_DIR = ROOT / 'mobilenet_v2_quantization'
+# Path to ImageNet validation dataset.
+DATASET_DIR = ROOT / 'imagenet'
+# Batch size
+BATCH_SIZE = 125
```

Comment:
Here is the batch size defined as a variable, but in the TensorFlow example above it was as a number. Also, here is the batch size is 125, and for TensorFlow is 128. Should this be aligned between the examples?

Reply:
Done